test/mlir/test/gc/Transforms/GPU/lit.local.cfg

```diff
@@ -1,2 +1,5 @@
 if not config.gc_use_imex:
-    config.unsupported = True
\ No newline at end of file
+    config.unsupported = True
+else:
+    # FIXME: Enable when #343 is merged.
+    config.excludes = ['gpu-to-gpuocl.mlir']
```

Comment:
The test should pass even without #343 since it's just the lowering, right?

Reply:
You are right, it should pass.